build.sbt

```diff
@@ -230,6 +234,22 @@ lazy val scodec = crossProject(JSPlatform, JVMPlatform)
 lazy val scodecJVM = scodec.jvm
 lazy val scodecJS = scodec.js
 
+lazy val scopt = crossProject(JSPlatform, JVMPlatform)
+  .configureCross(moduleCrossConfig("scopt"))
+  .dependsOn(core % "compile->compile;test->test")
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.github.scopt" %%% "scopt" % scoptVersion,
+      compilerPlugin(macroParadise % Test)
+    )
+  )
+
+lazy val scoptJVM = scopt.jvm
+lazy val scoptJS = scopt.js
+// This is required by scopt (which uses the 'os' modue), although I'm not 100% sure what other potential side effects
+// this might have.
+  .settings(scalaJSModuleKind := ModuleKind.CommonJSModule)
```

Comment:
I've no idea what this does, but can it be moved to `.jsSettings(...)` on the `lazy val scopt = ...`? I think to remember that no methods should be called after `.jvm` or `.js`.